[2025-07-15T22:08:22.316+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-15T22:08:22.370+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projeto_Joao_Motta.etapa_2 manual__2025-07-15T22:08:12.290733+00:00 [queued]>
[2025-07-15T22:08:22.386+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projeto_Joao_Motta.etapa_2 manual__2025-07-15T22:08:12.290733+00:00 [queued]>
[2025-07-15T22:08:22.387+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-15T22:08:22.435+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): etapa_2> on 2025-07-15 22:08:12.290733+00:00
[2025-07-15T22:08:22.458+0000] {standard_task_runner.py:63} INFO - Started process 59 to run task
[2025-07-15T22:08:22.467+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Projeto_Joao_Motta', 'etapa_2', 'manual__2025-07-15T22:08:12.290733+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/DAG 3.py', '--cfg-path', '/tmp/tmpzdx7e6f4']
[2025-07-15T22:08:22.473+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask etapa_2
[2025-07-15T22:08:22.562+0000] {task_command.py:426} INFO - Running <TaskInstance: Projeto_Joao_Motta.etapa_2 manual__2025-07-15T22:08:12.290733+00:00 [running]> on host fb8f5516f430
[2025-07-15T22:08:23.492+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projeto_Joao_Motta' AIRFLOW_CTX_TASK_ID='etapa_2' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T22:08:12.290733+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-15T22:08:12.290733+00:00'
[2025-07-15T22:08:23.494+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-15T22:08:23.718+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-15T22:08:23.719+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/DAG 3.py", line 63, in etapa_2
    df_final.to_excel(arquivo_saida, index=False)
  File "/home/airflow/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 2345, in to_excel
    formatter.write(
  File "/home/airflow/.local/lib/python3.10/site-packages/pandas/io/formats/excel.py", line 946, in write
    writer = ExcelWriter(  # type: ignore[abstract]
  File "/home/airflow/.local/lib/python3.10/site-packages/pandas/io/excel/_openpyxl.py", line 57, in __init__
    from openpyxl.workbook import Workbook
ModuleNotFoundError: No module named 'openpyxl'
[2025-07-15T22:08:23.764+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=Projeto_Joao_Motta, task_id=etapa_2, execution_date=20250715T220812, start_date=20250715T220822, end_date=20250715T220823
[2025-07-15T22:08:23.784+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task etapa_2 (No module named 'openpyxl'; 59)
[2025-07-15T22:08:23.809+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-15T22:08:23.829+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-15T22:08:23.832+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
